src/components/RefinementList/__tests__/__snapshots__/RefinementList-test.js.snap

```diff
@@ -264,7 +264,7 @@ exports[`RefinementList rendering with facets from search 1`] = `
       <form
         action=""
         className="ais-SearchBox-form"
-        noValidate="novalidate"
+        noValidate={true}
```

Comment:
apart from this PR, I find it weird to see snapshots of React components; and not of the real HTML that comes on the page. It might be hard to check if something is actually spelled the react-way or the regular way here

Reply:
That's true. We can change this in future tests. Which method would you recommend to use?